tools/bin/raiderDownloadGNSS.py

```diff
@@ -7,9 +7,9 @@
 #
 #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-from RAiDER.downloadGNSSDelays import parseCMD
+from RAiDER.downloadGNSSDelays import cmdLineParse,queryRepos
 
 if __name__ == "__main__":
+    inps = cmdLineParse()
 
-    # Main engine
-    parseCMD()
+    queryRepos(inps)
```

Comment:
Same comment as above

Reply:
Since ```queryRepos``` is a function though, shouldn't it follow this convention ```query_repos```

Reply:
This was in reference to the comment about unpacking arguments. But you are right, the convention would be to call it `query_repos` so that could be changed too.